src/main/scheduler/scheduler.c

```diff
@@ -494,7 +494,7 @@ FAST_CODE void scheduler(void)
             }
             DEBUG_SET(DEBUG_SCHEDULER_DETERMINISM, 0, clockCyclesTo10thMicros(cmpTimeCycles(nowCycles, lastTargetCycles)));
 #endif
-            currentTimeUs = clockCyclesToMicros(nowCycles);
+            currentTimeUs = micros();
```

Comment:
can you explain a little more on why this change is required, seems quite subtle.

Reply:
`clockCyclesToMicros` was causing overflows and regular peaks  with false positive signal loss.


```
int32_t clockCyclesToMicros(int32_t clockCycles)
{
    return clockCycles / usTicks;     // which is int32 / uint
}
```

max(clockCycles)=max(clockCycles) = 2147483647
usTicks = 216 for F7


2147483647/216 = 9 942 055 us = 10s roughly

Reply:
1) Better to ask @SteveCEvans . Generally as I said: false positive signal loss - each 20s. Probably bad timing of some tasks.
2) I have found regular 20s peaks in signal loss and then we arrange group with @ctzsnooze , @SteveCEvans  and @limonspb to resolve it. @SteveCEvans has found it.
3) I don't think so. @SteveCEvans has fixed this one.
4) I don't know if it can help with any other issue.
5) Yeah. That would be probably better solution.